app/src/main/java/org/greenstand/android/TreeTracker/userselect/UserSelectScreen.kt

```diff
@@ -71,22 +81,91 @@ fun UserSelectScreen(
             )
         }
     ) {
+        LazyVerticalGrid(
+            cells = GridCells.Fixed(2),
+            modifier = Modifier.padding(it),  // Padding for bottom bar.
+            contentPadding = PaddingValues(start = 8.dp, end = 8.dp, top = 10.dp)
+        ) {
+            items(state.planters) { user ->
+                UserButton(
+                    user = user,
+                    state = state,
+                ) { viewModel.selectUser(user) }
+            }
+        }
+    }
+}
 
+@Composable
+fun UserButton(
+    user: PlanterInfoEntity,
+    state: UserSelectState,
+    onClick: () -> Unit
+) {
+    val isSelected = state.selectedPlanter?.id == user.id
+
+    DepthButton(
+        onClick = onClick,
+        isSelected = isSelected,
+        modifier = Modifier
+            .size(width = 156.dp, height = 248.dp)
+            .padding(8.dp),
+        colors = DepthButtonColors(
+            color = AppColors.Gray,
+            shadowColor = if (isSelected) AppColors.GreenShadow else AppColors.GrayShadow,
+            disabledColor = AppColors.GrayShadow,
+            disabledShadowColor = AppColors.GrayShadow
+        )
+    ) {
         Column(
-            verticalArrangement = Arrangement.Center,
+            modifier = Modifier
+                .fillMaxHeight()
+                .fillMaxWidth()
+                .align(Alignment.TopCenter),
             horizontalAlignment = Alignment.CenterHorizontally
         ) {
-            Text("User Select")
-            state.planters.forEach { user ->
+            // User profile picture.
+            Box(  // TODO: Fetch user profile picture.
```

Comment:
To make this match parent you might be able to use fillMaxWidth and set its aspect ratio to 1:1